static/scss/share-popup.scss

```diff
@@ -46,6 +46,14 @@
   }
 }
```

Comment:
this file is styling both the share and login popups I think, what do you think about renaming it to just `popup.scss` while we're in here? I figure that's a little more clear

Reply:
👍